[2025-04-02T11:49:35.576+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-04-02T11:49:35.943+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: our_first_dag_2.first_task scheduled__2025-04-01T00:00:00+00:00 [queued]>
[2025-04-02T11:49:36.119+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: our_first_dag_2.first_task scheduled__2025-04-01T00:00:00+00:00 [queued]>
[2025-04-02T11:49:36.120+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 1
[2025-04-02T11:49:36.430+0000] {taskinstance.py:2890} INFO - Executing <Task(BashOperator): first_task> on 2025-04-01 00:00:00+00:00
[2025-04-02T11:49:36.436+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=216) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-04-02T11:49:36.438+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag_2', 'first_task', 'scheduled__2025-04-01T00:00:00+00:00', '--job-id', '199', '--raw', '--subdir', 'DAGS_FOLDER/fristdags.py', '--cfg-path', '/tmp/tmp66cjzxjk']
[2025-04-02T11:49:36.809+0000] {standard_task_runner.py:72} INFO - Started process 224 to run task
[2025-04-02T11:49:36.809+0000] {standard_task_runner.py:105} INFO - Job 199: Subtask first_task
[2025-04-02T11:49:37.835+0000] {task_command.py:467} INFO - Running <TaskInstance: our_first_dag_2.first_task scheduled__2025-04-01T00:00:00+00:00 [running]> on host d7b554d6843c
[2025-04-02T11:49:38.043+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='our_first_dag_2' AIRFLOW_CTX_TASK_ID='first_task' AIRFLOW_CTX_EXECUTION_DATE='2025-04-01T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-04-01T00:00:00+00:00'
[2025-04-02T11:49:38.044+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-04-02T11:49:38.958+0000] {subprocess.py:78} INFO - Tmp dir root location: /tmp
[2025-04-02T11:49:38.962+0000] {subprocess.py:88} INFO - Running command: ['/usr/bin/bash', '-c', "echo 'hello this first task'"]
[2025-04-02T11:49:38.973+0000] {subprocess.py:99} INFO - Output:
[2025-04-02T11:49:38.974+0000] {subprocess.py:106} INFO - hello this first task
[2025-04-02T11:49:38.975+0000] {subprocess.py:110} INFO - Command exited with return code 0
[2025-04-02T11:49:39.683+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-04-02T11:49:39.704+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=our_first_dag_2, task_id=first_task, run_id=scheduled__2025-04-01T00:00:00+00:00, execution_date=20250401T000000, start_date=20250402T114935, end_date=20250402T114939
[2025-04-02T11:49:40.949+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-04-02T11:49:41.099+0000] {taskinstance.py:3901} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-04-02T11:49:41.103+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
